web/src/components/countryhistorypricesgraph.jsx

```diff
@@ -20,7 +19,7 @@ import PriceTooltip from './tooltips/pricetooltip';
 const prepareGraphData = (historyData) => {
   if (!historyData || !historyData[0]) return {};
 
-  const currencySymbol = getSymbolFromCurrency(((first(historyData) || {}).price || {}).currency);
+  const currencySymbol = getSymbolFromCurrency(((historyData.at(0) || {}).price || {}).currency);
```

Comment:
~~Why use the [.at](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at) method instead of `[0]` - is there any benefits to that? :)~~ Nevermind, I see that you've used in other places together with `.at(-1)`, so it makes sense to just do that throughout :)